src/main/java/com/example/Restaurant_Management_System_REST_API/model/ContactDetails.java

```diff
@@ -1,18 +1,26 @@
 package com.example.Restaurant_Management_System_REST_API.model;
 
 import jakarta.persistence.Embeddable;
-import lombok.Getter;
-import lombok.Setter;
+import jakarta.validation.constraints.NotBlank;
+import lombok.*;
 
 @Getter
 @Setter
 @Embeddable
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
 public class ContactDetails {
-
+    @NotBlank(message = "Name must be provided.")
     private String name;
+    @NotBlank(message = "Street must be provided.")
     private String street;
+    @NotBlank(message = "House number must be provided.")
     private String houseNumber;
+    @NotBlank(message = "City must be provided.")
     private String city;
+    @NotBlank(message = "Postal code must be provided.")
     private String postalCode;
+    @NotBlank(message = "Telephone number must be provided.")
```

Comment:
I think it'd be nice to later make checks if all data is correctly inputed :)

Reply:
I got it!
done! :)